From: Jon Bernard <jbernard@debian.org>
Date: Wed, 15 Dec 2010 11:06:49 -0500
Subject: Replace misuse of hyphens in manpages with minus signs
To: libcg-devel@lists.sourceforge.net

Here is the relevant lintian information:

    This manual page seems to contain a hyphen where a minus sign was
    intended. By default, "-" chars are interpreted as hyphens (U+2010)
    by groff, not as minus signs (U+002D). Since options to programs use
    minus signs (U+002D), this means for example in UTF-8 locales that
    you cannot cut and paste options, nor search for them easily. The
    Debian groff package currently forces "-" to be interpreted as
    a minus sign due to the number of manual pages with this problem,
    but this is a Debian-specific modification and hopefully eventually
    can be removed.

    "-" must be escaped ("\-") to be interpreted as minus. If you really
    intend a hyphen (normally you don't), write it as "\(hy" to
    emphasise that fact. See groff(7) and especially groff_char(7) for
    details, and also the thread starting with
    http://lists.debian.org/debian-devel/2003/debian-devel-200303/msg01481.html

    If you use some tool that converts your documentation to groff
    format, this tag may indicate a bug in the tool. Some tools convert
    dashes of any kind to hyphens. The safe way of converting dashes is
    to convert them to "\-".

    Because this error can occur very often, Lintian shows only the
    first 10 occurrences for each man page and give the number of
    suppressed occurrences. If you want to see all warnings, run Lintian
    with the -d/--debug option.

    Refer to the groff_char(7) manual page for details.
---
 doc/man/cgclassify.1    |    2 +-
 doc/man/cgconfig.conf.5 |   14 +++++++-------
 doc/man/cgexec.1        |   12 ++++++------
 doc/man/cgget.1         |    8 ++++----
 doc/man/cgred.conf.5    |    6 +++---
 doc/man/cgrulesengd.8   |    4 ++--
 doc/man/lssubsys.1      |    6 +++---
 7 files changed, 26 insertions(+), 26 deletions(-)

diff --git a/doc/man/cgclassify.1 b/doc/man/cgclassify.1
index 2d8921b..b4815b8 100644
--- a/doc/man/cgclassify.1
+++ b/doc/man/cgclassify.1
@@ -6,7 +6,7 @@
 cgclassify \- move running task(s) to given cgroups
 
 .SH SYNOPSIS
-\fBcgclassify\fR [\fB-g\fR <\fIcontrollers>:<path\fR>] [--sticky | --cancel-sticky] <\fIpidlist\fR>
+\fBcgclassify\fR [\fB-g\fR <\fIcontrollers>:<path\fR>] [\-\-sticky | \-\-cancel-sticky] <\fIpidlist\fR>
 
 .SH DESCRIPTION
 this command moves processes defined by the list
diff --git a/doc/man/cgconfig.conf.5 b/doc/man/cgconfig.conf.5
index fea61ea..879c1ba 100644
--- a/doc/man/cgconfig.conf.5
+++ b/doc/man/cgconfig.conf.5
@@ -195,7 +195,7 @@ inside. It corresponds to following operations:
 .RS
 .nf
 mkdir /mnt/cgroups/cpu
-mount -t cgroup -o cpu,cpuacct cpu /mnt/cgroups/cpu
+mount \-t cgroup \-o cpu,cpuacct cpu /mnt/cgroups/cpu
 .fi
 .RE
 
@@ -272,7 +272,7 @@ It corresponds to following operations:
 .RS
 .nf
 mkdir /mnt/cgroups/cpu
-mount -t cgroup -o cpu,cpuacct cpu /mnt/cgroups/cpu
+mount \-t cgroup \-o cpu,cpuacct cpu /mnt/cgroups/cpu
 
 mkdir /mnt/cgroups/cpu/daemons
 
@@ -337,8 +337,8 @@ It corresponds to following operations:
 .nf
 mkdir /mnt/cgroups/cpu
 mkdir /mnt/cgroups/cpuacct
-mount -t cgroup -o cpu cpu /mnt/cgroups/cpu
-mount -t cgroup -o cpuacct cpuacct /mnt/cgroups/cpuacct
+mount \-t cgroup \-o cpu cpu /mnt/cgroups/cpu
+mount \-t cgroup \-o cpuacct cpuacct /mnt/cgroups/cpuacct
 
 mkdir /mnt/cgroups/cpu/daemons
 mkdir /mnt/cgroups/cpuacct/daemons
@@ -404,8 +404,8 @@ It corresponds to following operations:
 .nf
 mkdir /mnt/cgroups/cpu
 mkdir /mnt/cgroups/cpuacct
-mount -t cgroup -o cpu cpu /mnt/cgroups/cpu
-mount -t cgroup -o cpuacct cpuacct /mnt/cgroups/cpuacct
+mount \-t cgroup \-o cpu cpu /mnt/cgroups/cpu
+mount \-t cgroup \-o cpuacct cpuacct /mnt/cgroups/cpuacct
 
 mkdir /mnt/cgroups/cpuacct/daemons
 mkdir /mnt/cgroups/cpu/daemons
@@ -502,7 +502,7 @@ It corresponds to following operations:
 .RS
 .nf
 mkdir /mnt/cgroups/cpu
-mount -t cgroup -o cpu,cpuacct cpu /mnt/cgroups/cpu
+mount \-t cgroup \-o cpu,cpuacct cpu /mnt/cgroups/cpu
 
 chown root:operator /mnt/cgroups/cpu/*
 chown root:operator /mnt/cgroups/cpu/tasks
diff --git a/doc/man/cgexec.1 b/doc/man/cgexec.1
index bf197bd..2dc2812 100644
--- a/doc/man/cgexec.1
+++ b/doc/man/cgexec.1
@@ -7,7 +7,7 @@
 cgexec \- run the task in given control groups
 
 .SH SYNOPSIS
-\fBcgexec\fR [\fB-g\fR <\fIcontrollers>:<path\fR>] [--sticky] \fBcommand\fR [\fIarguments\fR]
+\fBcgexec\fR [\fB-g\fR <\fIcontrollers>:<path\fR>] [\-\-sticky] \fBcommand\fR [\fIarguments\fR]
 
 .SH DESCRIPTION
 The \fBcgexec\fR
@@ -46,7 +46,7 @@ changes the children tasks to the right cgroup based on
 
 .nf
 .ft B
-cgexec -g *:test1 ls
+cgexec \-g *:test1 ls
 .ft R
 .fi
 runs command \fBls\fR in control group test1
@@ -54,18 +54,18 @@ in all mounted controllers.
 
 .nf
 .ft B
-cgexec -g cpu,memory:test1 ls -l
+cgexec \-g cpu,memory:test1 ls \-l
 .ft R
 .fi
-runs command \fBls -l\fR in control group test1
+runs command \fBls \-l\fR in control group test1
 in controllers cpu and memory.
 
 .nf
 .ft B
-cgexec -g cpu,memory:test1 -g swap:test2 ls -l
+cgexec \-g cpu,memory:test1 \-g swap:test2 ls \-l
 .ft R
 .fi
-runs command \fBls -l\fR in control group test1
+runs command \fBls \-l\fR in control group test1
 in controllers cpu and memory and control group 
 test2 in controller swap.
 
diff --git a/doc/man/cgget.1 b/doc/man/cgget.1
index 39e8e36..24395ab 100644
--- a/doc/man/cgget.1
+++ b/doc/man/cgget.1
@@ -48,7 +48,7 @@ display help and exit
 
 .SH EXAMPLES
 .nf
-$ cgget -r cpuset.cpus -r cpuset.mems first second
+$ cgget \-r cpuset.cpus \-r cpuset.mems first second
 first:
 cpuset.cpus=0-1
 cpuset.mems=0
@@ -57,19 +57,19 @@ second:
 cpuset.cpus=0
 cpuset.mems=0
 
-$ cgget -n -r cpuset.cpus -r cpuset.mems first second
+$ cgget \-n \-r cpuset.cpus \-r cpuset.mems first second
 cpuset.cpus=0-1
 cpuset.mems=0
 cpuset.cpus=0
 cpuset.mems=0
 
-$ cgget -nv -r cpuset.cpus -r cpuset.mems first second
+$ cgget \-nv \-r cpuset.cpus \-r cpuset.mems first second
 0-1
 0
 0
 0
 
-$ cgget -n -g cpu /
+$ cgget \-n \-g cpu /
 cpu.rt_period_us=1000000
 cpu.rt_runtime_us=950000
 cpu.shares=1024
diff --git a/doc/man/cgred.conf.5 b/doc/man/cgred.conf.5
index 3fe760f..f4ee956 100644
--- a/doc/man/cgred.conf.5
+++ b/doc/man/cgred.conf.5
@@ -21,13 +21,13 @@ variable specifies the file to which logs will be written
 
 .TP
 \fBNODAEMON\fR
-if it is equal to "--nodamon" then
+if it is equal to "\-\-nodamon" then
 run cgred in non-daemon mode
 
 .TP
 \fBLOG\fR
-define the verbosity of logging. Accepted values are '-v', '-vv', '-q'
-and '--nolog'.
+define the verbosity of logging. Accepted values are '\-v', '\-vv', '\-q'
+and '\-\-nolog'.
 .LP
 
 
diff --git a/doc/man/cgrulesengd.8 b/doc/man/cgrulesengd.8
index 10d1d09..8febd54 100644
--- a/doc/man/cgrulesengd.8
+++ b/doc/man/cgrulesengd.8
@@ -48,13 +48,13 @@ messages and log errors only.
 Disable logging.
 .TP
 .B -d|--debug
-Equivalent to '-nvvf -', i.e. don't fork the daemon, display all log messages and
+Equivalent to '\-nvvf \-', i.e. don't fork the daemon, display all log messages and
 write them to the standard output.
 .TP
 .B -u <user>|--socket-user=<user>
 .B -g <group>|--socket-group=<group>
 Set owner of cgrulesengd socket. It assumes that cgexec tool runs with proper
-suid permissions so it can write to the socket when cgexec --sticky is used.
+suid permissions so it can write to the socket when cgexec \-\-sticky is used.
 .SH FILES
 .LP
 .PD .1v
diff --git a/doc/man/lssubsys.1 b/doc/man/lssubsys.1
index 2555cb9..f13821f 100644
--- a/doc/man/lssubsys.1
+++ b/doc/man/lssubsys.1
@@ -51,7 +51,7 @@ lists all subsystems which are present on the system,
 mounted ones have attached the mount point:
 .RS
 .nf
-$ lssubsys -am
+$ lssubsys \-am
 ns
 cpuset
 cpu,devices
@@ -69,7 +69,7 @@ mounted subsystems have attached the mount point,
 not mounted ones which are in some hierarchy have attached the hierarchy number:
 .RS
 .nf
-$ lssubsys -ami
+$ lssubsys \-ami
 ns
 cpuset 5
 cpu,devices 6
@@ -83,4 +83,4 @@ net_cls /cgroup/net_cls
 .RE
 .SH SEE ALSO
 lscgroup (1), cgcreate (1), cgdelete (1),
-cgconfig.conf (5)
\ No newline at end of file
+cgconfig.conf (5)
-- 
